warplib/src/conversions/integer_conversions.cairo

```diff
@@ -33,3 +36,34 @@ fn bool_into_felt252(val: bool) -> felt252 {
     let resp = OptionTraitImpl::<felt252>::unwrap(resp_option);
     resp
 }
+
+// Cairo-1.0 corelib code from the RC
+impl U256TryIntoFelt252 of TryInto<u256, felt252> {
+    fn try_into(self: u256) -> Option<felt252> {
+        let FELT252_PRIME_HIGH = 0x8000000000000110000000000000000_u128;
+        if self.high > FELT252_PRIME_HIGH {
+            return Option::None(());
+        }
+        if self.high == FELT252_PRIME_HIGH {
+            // since FELT252_PRIME_LOW is 1.
+            if self.low != 0 {
+                return Option::None(());
+            }
+        }
+        Option::Some(
+            self.high.into() * 0x100000000000000000000000000000000_felt252 + self.low.into()
+        )
+    }
+}
+
+fn unsafe_contract_address_from_u256(x: u256) -> ContractAddress {
+    match x.try_into() {
```

Comment:
`u256_to_felt252` is already defined in `utils.cairo` file

Reply:
Which `utils.cairo` are we talking about? The only `utils.cairo` I see is `warplib/src/maths/utils.cairo` and it's Cairo 0

Reply:
I did a global search of the project with ripgrep and there is no such function

Reply:
sorry I got confused with https://github.com/NethermindEth/warp/blob/5a0728efab84613fb06d546cab82129db927258d/warplib/src/maths/utils.cairo#L70

since it is in the pipeline